android/app/build.gradle

```diff
@@ -115,16 +115,13 @@ gradle.projectsEvaluated {
         android.applicationVariants.all { variant ->
             variant.outputs.each { output ->
                 output.processManifest.doLast {
-                    def f = new File(manifestOutputDirectory, 'AndroidManifest.xml')
-                    if (!f.isFile()) {
-                        f = new File(new File(manifestOutputDirectory, output.dirName), 'AndroidManifest.xml')
-                    }
-                    if (f.exists()) {
-                        def charset = 'UTF-8'
-                        def s = f.getText(charset)
-                        s = s.replace('</application>', "${dropboxActivity}</application>")
-                        f.write(s, charset)
-                    }
+                    def outputDir = manifestOutputDirectory.get().asFile
```

Comment:
so we don't need these checks anymore ? my understanding is that manifest should always be there and if it's not then it's better to fail that silently skip operation

Reply:
Correct. Actually this already hit me because with the update to Android Plugin 3.3 things change and this was being silently ignored. If it fails, let's fail loud!